src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,11 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
+    public String getRandomBall() {
+        Random random = new Random();
+        int qwe = random.nextInt(100);
+        return String.valueOf(qwe);
+    }
```

Comment:
The method 'getRandomBall' should return an instance of a Ball object with both a color and a number, not just a number as a String.